template/scripts/patch_liferay.sh

```diff
@@ -36,7 +36,7 @@ function main {
 		echo "[LIFERAY] Patching Tool updated successfully."
 	fi
 
-	if [ -e ${LIFERAY_PATCHING_DIR}/liferay-*.zip ]
+	if [ `ls ${LIFERAY_PATCHING_DIR}/liferay-*.zip | wc -l` -gt 0 ]
```

Comment:
Lets use the form `${}` instead of backticks ```` . The `${}` is the standard posix way.  

Also in your PR there seem to be some empty files added?  was that on purpose?

Reply:
When you talk about `${}`, do you mean `$()`?

Sorry about the empty files, they were not intended.